spec/inputs/tcp_spec.rb

```diff
@@ -269,6 +269,39 @@ def get_port
     end
   end
 
+  it "should flush codec after client disconnects" do
+    # verifies fix for https://github.com/logstash-plugins/logstash-input-tcp/issues/90
+    conf = <<-CONFIG
+      input {
+        tcp {
+          port => #{port}
+          codec => multiline {
+              pattern => "^\s"
+              what => "previous"
+          }
+        }
+      }
+    CONFIG
+
+    data = "a\n 1\n 2\nb\n 1"
+    event_count = 2
+
+    events = input(conf) do |pipeline, queue|
+      socket = Stud::try(5.times) { TCPSocket.new("127.0.0.1", port) }
+      socket.puts(data)
+      socket.close
+
+      event_count.times.collect {
```

Comment:
nit: multi-lines blocks should use `do` ... `end`. we try to use `{` ...  `}` for single-line blocks.